content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -58,7 +58,6 @@ different Kubernetes components.
 | `AllowInsecureBackendProxy` | `true` | Beta | 1.17 | |
 | `AnyVolumeDataSource` | `false` | Alpha | 1.18 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
-| `BalanceAttachedNodeVolumes` | `false` | Alpha | 1.11 | |
```

Comment:
Hi @ravisantoshgudimetla .
If the feature is removed and the feature gate is deprecated, you should move this entry to the Deprecated & GA table and add version 1.22.
See line ~202 in the Deprecated & GA table. Thanks.

Reply:
It is not even deprecated, we're removing it directly. I did not find any feature in that table that shows `Removed`. Do I just need to have entry which says

| `BalanceAttachedNodeVolumes` | `false` | Alpha | 1.11 | 1.22|


and no entry after it means, it never graduated to beta?

Reply:
In this context, deprecated means removed. Same goes for graduated, the feature gate is removed.